bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -104,6 +104,8 @@ struct ReplicaConfig {
   // throughput and number of messages sent.
   bool debugStatisticsEnabled = false;
 
+  // Metrics dump interval
+  uint64_t metricsDumpIntervalSeconds = 600;
```

Comment:
All ReplicaConfig parameters should have serialization/deserialization functions written in 
ReplicaConfigSerializer. The copy constructor needs to be updated as well.

Reply:
Done (although I'm not sure to which copy constructor you meant)